readme.md

```diff
@@ -0,0 +1,114 @@
+# Grazorator
+
+![gazorator logo](./assets/grazorator-horizontal-logo.png)
+
+Grazorator is a `dotnet` tool that empowers developers to generate XML based outputs using the power of the Razor View Engine.
```

Comment:
Gazorator is not a tool, it's a library which allows rendering Razor templates outside of ASP.NET Core.

Reply:
```suggestion
Grazorator is a `dotnet` library that empowers developers to generate XML based outputs using the power of the Razor View Engine.
```